storm-core/src/clj/org/apache/storm/daemon/supervisor.clj

```diff
@@ -425,6 +433,13 @@
          ". State: " state
          ", Heartbeat: " (pr-str heartbeat))
         (shutdown-worker supervisor id)))
+
+    (doseq [storm-id all-downloaded-storm-ids]
```

Comment:
@harshach Would I file a new issue regarding your suggestion, or do you want to file it?